client/app/bundles/Zoon/components/WorkflowDiagram.jsx

```diff
@@ -1,20 +1,28 @@
 import React from "react"
 import Reorder from "react-reorder"
+import classnames from "classnames"
+
+import {upFirstLetter} from "../utils"
 
 const families = ["occurrence", "covariate", "process", "model", "output"]
+export const allCompositionTypes = ["list", "chain", "replicate"]
```

Comment:
Because it would cause shadowing.

If the inconsistency bothers you (it does bother me a little, but not enough to sidetrack), I'd actually prefer to rename `families` to `allFamilies`.